contrib/typeshed/nacl/utils.pyi

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+class EncryptedMessage(bytes):
+    @property
+    def nonce(self): ...
+    @property
+    def ciphertext(self): ...
+
+class StringFixer: ...
+
+def bytes_as_string(bytes_in: Any): ...
+def random(size: int = ...): ...
```

Comment:
```suggestion
def random(size: int = ...) -> str: ...
```

Reply:
Mypy was complaining when encrypt() is called after this change but returning bytes instead seems to be working.